README.md

```diff
@@ -47,34 +52,123 @@ Clone the RPC-OpenStack repository:
 git clone https://github.com/rcbops/rpc-openstack /opt/rpc-openstack
 ```
 
+#### Run the basic system installation
+
 Start a screen or tmux session (to ensure that the deployment continues even
 if the ssh connection is broken) and run `deploy.sh`:
 
 Run the ``deploy.sh`` script within a tmux or screen session:
 
 ``` shell
-tmux
 cd /opt/rpc-openstack
-OSA_RELEASE="stable/pike" ./scripts/deploy.sh
+export RPC_PRODUCT_RELEASE="pike"  # This is optional, if unset the current stable product will be used
+./scripts/deploy.sh
 ```
 
+#### Configure and deploy the cloud
+
 To configure the installation please refer to the upstream OpenStack-Ansible
 documentation regarding basic [system setup](https://docs.openstack.org/project-deploy-guide/openstack-ansible/pike/configure.html).
 
-Prior to running the playbooks ensure your system(s) are using the latest
-artifacts. To ensure all hosts have the same artifacts run the RPC-OpenStack
-playbook `site-artifacts.yml`.
+Prior to running the OpenStack-Ansible playbooks ensure your system(s) are using
+the latest artifacts. To ensure all hosts have the same artifacts run the
+RPC-OpenStack playbook `site-artifacts.yml`.
+
+``` shell
+cd /opt/rpc-openstack
+export RPC_PRODUCT_RELEASE="pike"  # This is optional, if unset the current stable product will be used
```

Comment:
not sure if this should be it's own section, defining the release.  Since it's called out in at least 3 places so far.  Minor nit only though.

Reply:
While optional I'm attempting to make this painfully obvious.